tests/Requests/EncryptDecryptRequestTest.php

```diff
@@ -49,31 +49,4 @@ public function testInvalidDecryptAlgorithm()
         self::expectExceptionMessage('Invalid algorithm "bar"');
         new DecryptRequest('bar', 'foo');
     }
-
-    public function testInvalidEnvironmentSettingsMissingTenant()
-    {
-        putenv('AZURE_TENANT_ID=');
-        self::expectException(ClientException::class);
-        self::expectExceptionMessage('No suitable authentication method found.');
-        $authenticationFactory = new AuthenticatorFactory();
-        $authenticationFactory->getAuthenticator(new GuzzleClientFactory(new NullLogger()));
-    }
-
-    public function testInvalidEnvironmentSettingsMissingClient()
-    {
-        putenv('AZURE_CLIENT_ID=');
-        self::expectException(ClientException::class);
-        self::expectExceptionMessage('No suitable authentication method found.');
-        $authenticationFactory = new AuthenticatorFactory();
-        $authenticationFactory->getAuthenticator(new GuzzleClientFactory(new NullLogger()));
-    }
-
-    public function testInvalidEnvironmentSettingsMissingSecret()
-    {
```

Comment:
Tieto preco vyhadzujes?

Reply:
protoze to vubec netestuje EncryptDecryptRequest - dostalo se to tam nejak omylem, ty testy jsou v AuthenticationFactoryTest https://github.com/keboola/azure-key-vault-php-client/blob/bce10fe0a9bbcc72f478e291007d6bd75688e7d5/tests/Authentication/AuthenticationFactoryTest.php#L27-L58